expressmailing.php

```diff
@@ -98,13 +101,16 @@ public function install($alter_db = true)
 			&& $this->installAdminTab('AdminMarketingSList', false, $this->l('Sms-Mailing'), 'AdminMarketingS')
 			&& $this->installAdminTab('AdminMarketingEStats', false, $this->l('E-Mailing'), 'AdminMarketingE')
 			&& $this->installAdminTab('AdminMarketingFStats', false, $this->l('Fax-Mailing'), 'AdminMarketingF')
-			&& $this->installAdminTab('AdminMarketingSStats', false, $this->l('Sms-Mailing'), 'AdminMarketingS');
+			&& $this->installAdminTab('AdminMarketingSStats', false, $this->l('Sms-Mailing'), 'AdminMarketingS')
+			/* Add QuickLink */
+			&& $this->installQuickAccess();
 	}
 
 	public function uninstall($alter_db = true)
 	{
 		return parent::uninstall()
 			&& Configuration::deleteByName('adminmarketing_session_api')
+			&& $this->uninstallQuickAccess()
```

Comment:
Because you should not insert your link into the quick access, this line should be removed as well.